### PR TITLE
fix: MD006/ul-start-left

### PR DESCRIPTION
**Pull request by nschonni**
Consider starting bulleted lists at the beginning of the line

### CODE REVIEW

**Review by Trott — APPROVED** [review]
Rubber-stamp LGTM

File: locale/en/blog/release/v0.12.0.md [hunk 1]
```diff
@@ -16,15 +16,15 @@ Node.js has seen many changes between v0.10 and v0.12. There is [a listing of
 changes documented on the wiki](https://github.com/joyent/node/wiki/API-changes-between-v0.10-and-v0.12). Note that this release includes API updates
 that may require dependency updates. Following are some highlights:
 
- * Streams 3
+* Streams 3
```
Comment (nschonni) [inline locale/en/blog/release/v0.12.0.md:19]: This wasn't rendering as a parent LI before

File: locale/en/blog/release/v11.10.0.md [hunk 2]
```diff
@@ -10,18 +10,18 @@ author: Michaël Zasso
 
 ### Notable Changes
 
-**deps**
+* **deps**
```
Comment (nschonni) [inline locale/en/blog/release/v11.10.0.md:13]: Made these list items like some of the other release notes

File: locale/en/blog/release/v4.0.0.md [hunk 3]
```diff
@@ -36,10 +36,10 @@ This list of changes is relative to the last io.js v3.x branch release, v3.3.0.
...
- - Implement new `TypedArray` prototype methods: `copyWithin()`, `every()`, `fill()`, `filter()`, `find()`, `findIndex()`, `forEach()`, `indexOf()`, `join()`, `lastIndexOf()`, `map()`, `reduce()`, `reduceRight()`, `reverse()`, `slice()`, `some()`, `sort()`. See https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/TypedArray for further information.
- - Implement new `TypedArray.from()` and `TypedArray.of()` functions. See https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/TypedArray for further information.
- - Implement arrow functions, see https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Functions/Arrow_functions for further information.
- - Full ChangeLog available at https://github.com/v8/v8-git-mirror/blob/4.5.103/ChangeLog
+  - Implement new `TypedArray` prototype methods: `copyWithin()`, `every()`, `fill()`, `filter()`, `find()`, `findIndex()`, `forEach()`, `indexOf()`, `join()`, `lastIndexOf()`, `map()`, `reduce()`, `reduceRight()`, `reverse()`, `slice()`, `some()`, `sort()`. See https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/TypedArray for further information.
```
Comment (nschonni) [inline locale/en/blog/release/v4.0.0.md:39]: These weren't rendering under the parent LI

File: locale/en/blog/release/v5.10.1.md [hunk 4]
```diff
@@ -10,10 +10,10 @@ author: Myles Borins
 
 ### Notable changes
 
-**http**:
+* **http**:
```
Comment (nschonni) [inline locale/en/blog/release/v5.10.1.md:13]: Made these parent LIs like other release notes

File: locale/en/blog/release/v5.9.0.md [hunk 5]
```diff
@@ -13,13 +13,13 @@ author: Evan Lucas
...
 https://github.com/nodejs/node/pull/5392
 * **governance**: The following members have been added as collaborators:
- - Andreas Madsen (@AndreasMadsen)
- - Benjamin Gruenbaum (@benjamingr)
- - Claudio Rodriguez (@claudiorodriguez)
- - Glen Keane (@thekemkid)
- - Jeremy Whitlock (@whitlockjc)
- - Matt Loring (@matthewloring)
- - Phillip Johnsen (@phillipj)
+  - Andreas Madsen (@AndreasMadsen)
```
Comment (nschonni) [inline locale/en/blog/release/v5.9.0.md:16]: These weren't rendering under the parent LI

File: locale/en/blog/weekly-updates/weekly-update.2015-09-04.md [hunk 6]
```diff
@@ -28,10 +28,10 @@ This list of changes is relative to the last io.js v3.x branch release, v3.3.0.
...
- - Implement new `TypedArray` prototype methods: `copyWithin()`, `every()`, `fill()`, `filter()`, `find()`, `findIndex()`, `forEach()`, `indexOf()`, `join()`, `lastIndexOf()`, `map()`, `reduce()`, `reduceRight()`, `reverse()`, `slice()`, `some()`, `sort()`. See https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/TypedArray for further information.
- - Implement new `TypedArray.from()` and `TypedArray.of()` functions. See https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/TypedArray for further information.
- - Implement arrow functions, see https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Functions/Arrow_functions for further information.
- - Full ChangeLog available at https://github.com/v8/v8-git-mirror/blob/4.5.103/ChangeLog
+  - Implement new `TypedArray` prototype methods: `copyWithin()`, `every()`, `fill()`, `filter()`, `find()`, `findIndex()`, `forEach()`, `indexOf()`, `join()`, `lastIndexOf()`, `map()`, `reduce()`, `reduceRight()`, `reverse()`, `slice()`, `some()`, `sort()`. See https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/TypedArray for further information.
```
Comment (nschonni) [inline locale/en/blog/weekly-updates/weekly-update.2015-09-04.md:31]: This wasn't rendering under the parent LI

File: tests/scripts/CHANGELOG.fixture.md [hunk 7]
```diff
@@ -1589,10 +1589,10 @@ This list of changes is relative to the last io.js v3.x branch release, v3.3.0.
...
- - Implement new `TypedArray` prototype methods: `copyWithin()`, `every()`, `fill()`, `filter()`, `find()`, `findIndex()`, `forEach()`, `indexOf()`, `join()`, `lastIndexOf()`, `map()`, `reduce()`, `reduceRight()`, `reverse()`, `slice()`, `some()`, `sort()`. See https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/TypedArray for further information.
- - Implement new `TypedArray.from()` and `TypedArray.of()` functions. See https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/TypedArray for further information.
- - Implement arrow functions, see https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Functions/Arrow_functions for further information.
- - Full ChangeLog available at https://github.com/v8/v8-git-mirror/blob/4.5.103/ChangeLog
+  - Implement new `TypedArray` prototype methods: `copyWithin()`, `every()`, `fill()`, `filter()`, `find()`, `findIndex()`, `forEach()`, `indexOf()`, `join()`, `lastIndexOf()`, `map()`, `reduce()`, `reduceRight()`, `reverse()`, `slice()`, `some()`, `sort()`. See https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/TypedArray for further information.
```
Comment (nschonni) [inline tests/scripts/CHANGELOG.fixture.md:1592]: This wasn't rendering under the parent LI

**Review by nschonni — COMMENTED** [review]
Sorry this is noisy. I added comments to the parts that caused the "rich diff" to show rendering changes. The other files changes do not affect output

**Review by Aissaoui-Ahmed — APPROVED** [review]

**Pull request merged**